changelog/13620.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
We don't need to add a changelog for developer happiness additions (and can just mention that in the PR to explain why there isn't one)

Reply:
Ok thanks that's good to know I'll remove it.